app/models/agents/http_agent.rb

```diff
@@ -46,22 +47,25 @@ def validate_options
     end
 
     def check
-      check_this_url interpolated[:url]
+      check_this_url interpolated[:url] interpolated[:header]
```

Comment:
You'll need a comma after `interpolated[:url]` to separate the method arguments.